modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -23,44 +23,31 @@ The name of your secret group must:
 
-== Add a Keystore
+[[add_certificate]]
+== Add a Certificate
 
-Specify the type of keystore to add to the secret group. 
+Add a single PEM certificate to the secret group. This certificate is the "leaf" certificate (end-entity certificate). 
 
-. In the *Secret Groups* list view, select the secret group to which to add a keystore. 
-. Select *Keystore* in the menu on the left, and click *Add Keystore*. 
-. In the *Name* field, enter a name for your keystore.
-. In the *Type* field, select the keystore type from the drop-down menu.
-+
-Supported types include the following:
+. In the *Secret Groups* list view, select the secret group to which to add a certificate. 
+. Select *Certificate* in the menu on the left, and click *Add Certificate*. 
+. In the *Add Certificate* screen, add the required information. 
 +
-* Privacy Enhanced Mail (PEM) - Base64 encoded ASCII file with a `cer`, `crt`, or `pem` extension.
-* Java Keystore (JKS) - Repository for authorization or public key certificates. The JKS keystore type does not store secret keys.
-* PKCS12 - Stores server and intermediate certificates in an archive file format. The PKCS12 keystore type does not store secret keys.
-* Java Cryptography Extension KeyStore (JCEKS) - Stores server and intermediate certificates as well as secret keys.
-.. To add a *PEM* type keystore, you must provide the following: 
+* *Name* - Enter a name for your certificate.
+* *Type* -  Select the certificate type from the drop-down menu.
 +
-** *Certificate File* - Click *Choose File* to locate and upload the PEM certificate file.  
-** *Key File* - Click *Choose File* to locate and upload the PEM formatted file that contains the private key for the certificate. 
-** *Key Passphrase* - Enter the word or phrase that protects the private key. 
-** *CA Path Certificate File* (Optional) - Click *Choose File* to locate and upload the certificate signed by a certification authority.   
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
-.. To add a *JKS*, *PKCS12*, or *JCEKS* type keystore, you must provide the following: 
+Supported types are:
 +
-** *Keystore File* - Click *Choose File* to locate and upload the keystore file to use.
-** *Keystore Passphrase* - Enter the word or phrase that protects the keystore.
-** *Alias* - The alias used to access the keystore entries (key and trusted certificate entries).
-** *Key Passphrase* - The word or phrase that protects the private key.
-** *Algorithm* (Optional) - The algorithm to use for encryption of keys.
-** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+** PEM
+* *Certificate File* -  Click *Choose File*, and select the certificate file to upload.
+* *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
```

Comment:
I don't understand the bullet nesting here---should cert file and override exp date be at the same level as PEM? Are they part of a different list? Maybe PEM should be one bullet not two?

Reply:
Verified the bullet nesting, and it's as intended.